### PR TITLE
LINK-2065 | add 25.5% as new vat option

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Add 25.5% as new vat option

**NOTE! This can only be merged in August before the production testing.**

## Closes
[LINK-2065](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2065)

[LINK-2065]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2065?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr416-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr416-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by harriris-vincit — APPROVED** [review]
One comment about the label value of the new VAT %, but otherwise looks really good.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr416-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/domain/priceGroup/hooks/useVatOptions.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { OptionType } from '../../../types';
 
-const VAT_PERCENTAGES = [24, 14, 10, 0];
+const VAT_PERCENTAGES = [25.5, 24, 14, 10, 0];
 
 const useVatOptions = (): OptionType[] => {
   return VAT_PERCENTAGES.map((vat) => ({
-    label: `${vat} %`,
-    value: `${vat}.00`,
+    label: `${vat.toFixed(2)} %`,
```
[the remark this answers is not in the document]
Comment (jorilindell) [reply src/domain/priceGroup/hooks/useVatOptions.ts]: I changed label to use `toLocaleString` instead of fixed decimals.

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr416-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by harriris-vincit — CHANGES_REQUESTED** [review]
After recent discussions and planning, the 24 % VAT option should be removed completely.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr416-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr416-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Review by harriris-vincit — APPROVED** [review]
@mikkojamG LGTM :+1: This can be merged now since the 25,5 % support has just been added to the back-end as well.

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr416-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**